cypress/integration/tests/developer/controls/businessservices/pagination.test.ts

```diff
@@ -17,104 +17,71 @@ limitations under the License.
 
 import {
     login,
-    clickByText,
+    // clickByText,
     selectItemsPerPage,
     deleteTableRows,
     preservecookies,
     hasToBeSkipped,
     createMultipleBusinessServices,
     deleteAllBusinessServices,
-    selectUserPerspective,
+    // selectUserPerspective,
+    deleteByList,
+    goToLastPage,
+    deleteFromArray,
+    validatePagination,
 } from "../../../../../utils/utils";
-import { navMenu, navTab } from "../../../../views/menu.view";
-import { controls, businessServices } from "../../../../types/constants";
+// import { navMenu, navTab } from "../../../../views/menu.view";
+import {
+    // controls, businessServices,
+    SEC,
+} from "../../../../types/constants";
 import {
     firstPageButton,
     lastPageButton,
     nextPageButton,
     pageNumInput,
     prevPageButton,
 } from "../../../../views/common.view";
+import { BusinessServices } from "../../../../models/developer/controls/businessservices";
 
 describe("Business services pagination validations", { tags: "@tier3" }, function () {
+    let businessServiceList = [];
     before("Login and Create Test Data", function () {
         // Prevent hook from running, if the tag is excluded from run
         if (hasToBeSkipped("@tier3")) return;
 
         // Perform login
         login();
         // Clear pre-existing data
-        deleteAllBusinessServices();
+        // deleteAllBusinessServices();
         // Create 11 rows
-        createMultipleBusinessServices(11);
+        businessServiceList = createMultipleBusinessServices(11);
     });
 
     beforeEach("Persist session", function () {
         // Save the session and token cookie for maintaining one login session
         preservecookies();
 
         // Interceptors for business services
-        cy.intercept("GET", "/hub/business-service*").as("getBusinessService");
-    });
-
-    after("Perform test data clean up", function () {
-        // Prevent hook from running, if the tag is excluded from run
-        if (hasToBeSkipped("@tier3")) return;
-
-        // Delete the business services created before the tests
-        deleteAllBusinessServices();
+        // cy.intercept("GET", "/hub/business-service*").as("getBusinessService");
```

Comment:
Comments were to be removed. @ibragins Let's merge PR only if it is approved by someone else.